docs/widgets.md

```diff
@@ -22,6 +22,23 @@ Widgets define the data type and interface for entry fields. Netlify CMS comes w
 
 We’re always adding new widgets, and you can also [create your own](/docs/extending)!
 
+### Select Widget
+
+The select widget allows you to pick a string value from a drop down menu
+
+```yaml
+collections:
+  - name: posts
+    label: Post
+    folder: "_posts"
+    slug: "{{year}}-{{month}}-{{day}}-{{slug}}"
+    create: true
+    fields:
+      - {label: Title, name: title, widget: string, tagname: h1}
+      - {label: Body, name: body, widget: markdown}
+      - {label: Align Content, name: align, widget: select, options: ['left', 'center', 'right']}
```

Comment:
Closing #764 requires documentation of adding labels and values, would you mind adding an example of that, too? You could even trim this code sample down and just show the field config examples, e.g.:

```yaml
collections:
  - name: posts
     ...
     fields:
       ...
       - { label: Align Content, <rest of field> }
       - { label: Other example, <rest of field> }
```